kernel/packages/shared/meta/sagas.ts

```diff
@@ -29,6 +30,22 @@ export function* metaSaga(): any {
 
   yield put(metaConfigurationInitialized(config))
   yield call(checkExplorerVersion, config)
+  yield call(checkIndexedDB, config)
+}
+
+function checkIndexedDB(config: Partial<MetaConfiguration>) {
+  if (!config || !config.explorer) {
+    return
+  }
+
+  if (!config || !config.explorer || !config.explorer.useUnityIndexedDbCache) {
```

Comment:
`config` and `config.explorer` have already been checked in the previous conditional.